packages/rum/src/domain/segmentCollection/segmentCollection.ts

```diff
@@ -2,13 +2,13 @@ import type { EventEmitter, TimeoutId } from '@datadog/browser-core'
 import { addEventListener, DOM_EVENT, monitor } from '@datadog/browser-core'
 import type { LifeCycle, ViewContexts, RumSessionManager } from '@datadog/browser-rum-core'
 import { LifeCycleEventType } from '@datadog/browser-rum-core'
-import { SEND_BEACON_BYTE_LENGTH_LIMIT } from '../../transport/send'
+import { SEND_BEACON_BYTES_LIMIT } from '../../transport/send'
 import type { CreationReason, Record, SegmentContext, SegmentMetadata } from '../../types'
 import type { DeflateWorker } from './deflateWorker'
 import { Segment } from './segment'
 
-export const MAX_SEGMENT_DURATION = 30_000
-let MAX_SEGMENT_SIZE = SEND_BEACON_BYTE_LENGTH_LIMIT
+export const SEGMENT_DURATION_LIMIT = 30_000
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
export const SEGMENT_DURATION_LIMIT = 30 * ONE_SECOND
```